receiver/k8sclusterreceiver/metadata.yaml

```
@@ -497,3 +507,19 @@ metrics:
 
   # k8s.node.allocatable_* metrics (k8s.node.allocatable_cpu, k8s.node.allocatable_memory, etc) are controlled
   # by allocatable_types_to_report config option. By default, none of them are reported.
+  k8s.hierarchical_resource_quota.hard_limit:
```

```suggestion
  k8s.hierarchicalresourcequota.limit:
```

This will keep the metric names in-line with the attr names which do not have underscore. Are we adding `hard_limit` explicitly cause there is another type of limit with these (e.g. soft_limit)?

---

I refer to the following lines.
Should I change that setting to 'limit'?

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/6a0b248b6bfdbc86c8cc856b3d91861de06efec4/receiver/k8sclusterreceiver/metadata.yaml#L418-L433